toolchain/check/import_cpp.cpp

```diff
@@ -44,6 +44,16 @@ static auto GenerateCppIncludesHeaderCode(
   std::string code;
   llvm::raw_string_ostream code_stream(code);
   for (const Parse::Tree::PackagingNames& import : imports) {
+    // Add a line marker directive pointing at the location of the `import Cpp`
+    // declaration in the Carbon source file. This will cause Clang's
+    // diagnostics machinery to track and report the location in Carbon code
+    // where the import was written.
+    auto import_loc = context.tokens().TokenToDiagnosticLoc(
```

Comment:
TokenToDiagnosticLoc seems like it does a bit of extra work for diagnostic structure, like column and length. Could this be using `tokens().GetLineNumber` and `tokens().source().filename()` given the limited amount it uses?

Reply:
Done.